react/Image.tsx

```diff
@@ -82,6 +83,9 @@ function Image(props: ImageProps) {
     promotionPosition,
     classes,
     preload,
+    // Do not use this property
```

Comment:
Why?

Reply:
Because this property is internal, to handle when the Image component is inside the slider-layout

Reply:
You shouldn't use this through the blocks. Because of that, I didn't add this prop to the documentation.

Reply:
I will write a better explanation